### ISSUE TITLE
[Bug]: Failed to connect to docker proxy

### ISSUE THREAD

**Issue by iamjaredwalters** [livecycle/preevy#218]
### What happened?

Unsure of root cause but receive this message periodically:

```
Getting tunnel URLs...
    Error: Failed to connect to docker proxy at https://preevy_proxy-platform-
    feat-pr-review-stacks-d52axemz.livecycle.run/: 500: Internal Server Error
``` 

Using 
```
preevy version
0.0.50
```

Any tips on how to debug? It's always after an environment exists (successful deploys) but beyond that Im not sure what triggers the issue nor how to resolve. So far, I've destroyed the instance manually and re-ran preevy up but the problem persists. Will update as I discover more but would appreciate any help diagnosing in the mean time.

### Add screenshots

Some more output in case useful

```
#66 DONE 0.1s
 Network platform_default  Creating
 Network platform_default  Created
 Volume "platform_storage"  Creating
 Volume "platform_storage"  Created
 Volume "platform_postgres"  Creating
 Volume "platform_postgres"  Created
 Container platform-db  Creating
 Container platform-preevy_proxy-1  Creating
 Container platform-frontend  Creating
 Container platform-redis  Creating
 Container platform-db  Created
 Container platform-frontend  Created
 Container platform-redis  Created
 Container platform-worker  Creating
 Container platform-preevy_proxy-1  Created
 Container platform-backend  Creating
 Container platform-worker  Created
 Container platform-backend  Created
 Container platform-preevy_proxy-1  Starting
 Container platform-db  Starting
 Container platform-frontend  Starting
 Container platform-redis  Starting
 Container platform-db  Started
 Container platform-preevy_proxy-1  Started
 Container platform-redis  Started
 Container platform-worker  Starting
 Container platform-backend  Starting
 Container platform-frontend  Started
 Container platform-backend  Started
 Container platform-worker  Started
Preview environment platform-feat-pr-review-stacks provisioned at: 35.91.245.124
- Getting tunnel URLs...
✖ Getting tunnel URLs...
    Error: Failed to connect to docker proxy at https://preevy_proxy-platform-
    feat-pr-review-stacks-d52axemz.livecycle.run/: 500: Internal Server 
```

### Steps to reproduce the behavior

Not yet but will provide if I can determine them

### Expected behavior

`preevy up` reproduces the environment without issue

### What OS are you seeing the problem on?

Linux

### Additional context

Using AWS Lightsail

### Record

- [X] I agree to follow this project's Code of Conduct

**Comment by iamjaredwalters** [comment]
A little more info, `preevy down`, `preevy purge`, and manually deleting the instance from AWS UI doesn't seem to resolve the error which leads me to suspect its on the preevy side of the fence.

**Comment by iamjaredwalters** [comment]
`preevy ls` output

```bash
preevy ls
 Env Driver ID Location State
 ─── ───────── ──────── ─────
```

**Comment by Yshayy** [comment]
Hey @iamjaredwalters, thanks for reporting the issue.
It seems we have a problem with zombie tunnels (that's why using a new --id did the trick), we're working on understanding the root cause and apply a fix.

**Comment by Yshayy** [comment]
Fixed in #227
